### ISSUE TITLE
Audit and improve alerts

### ISSUE THREAD

**Issue by comaurice** [department-of-veterans-affairs/va.gov-team#97029]
## Value Statement

**_As a_** Dev

**_I want to_** streamline all the DR alerts that we currently generate

**_So that_** I have clarity on the important slack alerts

---

## Background Context

- We have a lot of alert monitors set up in DataDog, and also alerts that come through a few other channels. Can we consolidate, or reduce noise?

## Outcome, Success Measure, KPI(S), and Tracking Link

Every alert that appears in our Slack channel requires action or awareness within 24 hours

## Acceptance Criteria

- [ ] Updated playbooks for alerts
- [ ] Clear action for each alert or warning

## Designs and Build Notes

- Add here

## Tasks

- [ ] List of alerts, including source and delivery mechanism
- [ ] Can we use our new metrics to get better alerts less often?

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] New playbooks shared
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by shaunburdick** [comment]
Putting this on hold until we get the results of the All Hands discussion on error categorization. Steve Albers started a [document](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/benefits/architectural-decision-records/more-actionable-alerts.md) on error classification and SLOs and we will need to reconnect before starting this work.

**Comment by comaurice** [comment]
Following the All hands meeting, not much was gained. Will move to auditing alerts and ensuring they're actionable, convert to -> non alert.

**Comment by dfong-adh** [comment]
Created monitor https://vagov.ddog-gov.com/monitors/320480 to track SavedClaim sidekiq job errors.

**Comment by eileen-coforma** [comment]
Need to review remaining monitors and look for improvements

**Comment by dfong-adh** [comment]
Added APM trace queries to ContestableIssue monitors:
https://vagov.ddog-gov.com/monitors/215140
https://vagov.ddog-gov.com/monitors/202306
https://vagov.ddog-gov.com/monitors/216094

**Comment by dfong-adh** [comment]
Job monitor updated to reflect latest job changes:
https://vagov.ddog-gov.com/monitors/290117
